packages/console/src/pages/Users/components/CreateForm/index.tsx

```diff
@@ -29,7 +29,7 @@ const CreateForm = ({ onClose }: Props) => {
     try {
       const createdUser = await api.post('/api/users', { json: data }).json<User>();
 
-      onClose?.(createdUser);
+      onClose?.(createdUser, btoa(data.password));
     } catch (error: unknown) {
       console.error(error);
```

Comment:
Add a TODO here? feels like the request exception needs to be handled.

Reply:
my fault, I just copied those lines of code, the try catch is no longer needed, `api` will handle general errors.
updated, PTAL